Formula/zilf.rb

```diff
@@ -0,0 +1,28 @@
+class Zilf < Formula
+  desc "Tools for working with the Zork Implementation Language from Infocom"
+  homepage "https://bitbucket.org/jmcgrew/zilf/"
+  url "https://bitbucket.org/jmcgrew/zilf/downloads/zilf-0.8.zip"
+  sha256 "60b38260e868910095a6351e5c635ebf58e12dbd66de9677665664ccbd603286"
+
+  bottle :unneeded
+
+  depends_on "mono"
+
+  def install
+    prefix.install "README.txt"
+    bin.install Dir["bin/*.dll", "bin/*.exe"]
+    prefix.install Dir["doc", "library", "sample"]
+    (bin/"zilf").write <<~EOS
+      #!/bin/sh
+      mono #{bin}/zilf.exe "$@"
```

Comment:
>I wasn't able to get zilf to compile the samples in any other directory then "bin/" (just unzipping it locally and renaming the folder means some includes start to break)

You can pass the include path explicitly with `-ip`:

```suggestion
      mono #{bin}/zilf.exe -ip "#{prefix}/library" "$@"
```